index.d.ts

```diff
@@ -134,7 +134,7 @@ export interface PopperChildren {
 export interface HideAllOptions {
   checkHideOnClick?: boolean
   duration?: number
-  exclude?: Instance
+  exclude?: any
 }
```

Comment:
I don't know how to type this to work with the new `hideAll` behavior, cc @KubaJastrz